src/build/spawn_stream.mjs

```diff
@@ -25,26 +64,24 @@ export const spawnStream = (command, ...parameters) =>
     const stderr = [];
 
     console.debug(chalk.gray(`Running [${[command, ...parameters.map(e => `'${e}'`)].join(' ')}]...`));
-    const childProcess = spawn(command, parameters, {env: process.env});
-
-    const forEachMessageLine = (buffer, callback) => {
-      buffer
-        .toString()
-        .split('\n')
-        .filter(line => line.trim())
-        .forEach(callback);
-    };
+    const childProcess = spawn(command, parameters, {env: process.env, stdio: ['inherit', 'pipe', 'pipe']});
 
-    childProcess.stdout.on('data', data =>
-      forEachMessageLine(data, line => {
-        console.info(line);
-        stdout.push(line);
-      })
-    );
+    const stdOutPipe = newChildProcessOutputPipeTransform(line => {
+      console.info(line);
+      stdout.push(line);
+    });
+    childProcess.stdout.pipe(stdOutPipe);
 
-    childProcess.stderr.on('data', error => forEachMessageLine(error, line => stderr.push(line)));
+    const stdErrPipe = newChildProcessOutputPipeTransform(line => {
```

Comment:
The problem is `stderr` doesn't always mean error. For example, in the case of the taskfile, it outputs normal content to `stderr` (see the image I attached above). If we prepend `[STDERR]` to each line, it might actually cause confusion.

Reply:
Yeah, so I think the question is why do we want to distinguish from these two streams? If you run a command in the terminal, it won't tell any difference. The command will use colors or log types like `ERROR: xxx` to tell the user whether there are any errors.

Reply:
Let's take an example of this output: 
<img width="809" alt="image" src="https://github.com/Jigsaw-Code/outline-apps/assets/93548144/acb31133-15d5-41da-a816-ad26a94aa784">

The first three lines are from `stdout`, and all the following are from `stderr`. Do you prefer we output something like the following?

```
[stdout] Running [node '--trace-uncaught' ...
[stdout] Running [go 'run' ...
[stdout] task: [/home/runner/work/...]
[stderr] task: "client:tun2socks:linux" started
[stderr] task: "client:tun2socks:electron" started
 ...
[stderr]
[stderr] task "client:tun2socks:electron" finished
[stderr] task "client:tun2socks:linux" finished
```

Reply:
You need to differentiate info output from the Task app from the actual task output. That's why the difference.